### DISCUSSION TITLE
Phising Website

### DISCUSSION THREAD

**Discussion by denisanjayaaa** [vercel/community#3547] [Bug Report]
### Page to Investigate

https://danaidkagett2023-1.vercel.app/

### Steps to Reproduce

https://danaidkagett2023-1.vercel.app/

**Comment by amyegan** [comment]
That site is disabled now. Thanks for reporting.

**Comment by denisanjayaaa** [comment]
Phising Website
https://danafsk-prioritas.vercel.app/

Pada tanggal Rab, 9 Agu 2023 21.19, Amy ***@***.***> menulis:

**Reply by amyegan** [reply]
If you believe that a Vercel deployment contains malicious or illegal content, please email [abuse@vercel.com](mailto:abuse@vercel.com) where the request will be dealt with immediately. When doing this, please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) and [trademark policy](https://vercel.com/legal/trademark-policy).
